addons/mail/data/ir_sequence_data.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <!-- Sequences for auto generated mail aliases -->
+        <record id="seq_mail_alias" model="ir.sequence">
```

Comment:
Or not...

Reply:
Sorry, I changed to draft because I was about to change things, the current code was not meant to be ready for review